api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestSingleShot.java

```diff
@@ -945,4 +945,44 @@ public void testSNPEClassificationResult() {
             fail();
         }
     }
+
+    @Test
+    public void testPytorchClassificationResult() {
+        if (!NNStreamer.isAvailable(NNStreamer.NNFWType.PYTORCH)) {
+            /* cannot run the test */
+            return;
+        }
+
+        /* expected label is orange (950) */
+        final int expected_label = 950;
+
+        try {
+            File model = APITestCommon.getPytorchModel();
+
+            TensorsInfo in = new TensorsInfo();
+            in.addTensorInfo("", NNStreamer.TensorType.FLOAT32, new int[]{224, 224, 3, 1});
+
+            TensorsInfo out = new TensorsInfo();
+            out.addTensorInfo("", NNStreamer.TensorType.FLOAT32, new int[]{1000, 1});
+
+            SingleShot single = new SingleShot(new File[]{model}, in, out, NNStreamer.NNFWType.PYTORCH, "");
+
+            /* invoke */
+            TensorsData output = single.invoke(APITestCommon.readRawImageDataPytorch());
+
+            /* let's ignore timeout (set 60 sec) */
+            single.setTimeout(60000);
```

Comment:
this line is useless after invoke is done.

Reply:
Oh my bad.. It's removed. Thanks!